docs/source/yelpsoa_configs.rst

```diff
@@ -17,6 +17,9 @@ specify the following options:
     available free CPU, but is guaranteed to get the CPU shares specified.  For
     a more detailed read on how this works in practice, see the docs on `isolation <isolation.html>`_.
 
+    * ``cpu_burst_add``: Number of additional CPUs an instance may use while
+      bursting; if unspecified, PaaSTA defaults to 1.
```

Comment:
Say a bit more, `which means if a serivce ask for `cpus: 1`, it can actually burst to 2 full cpus if needed.